content/news/buzz/october-2023-status-update/contents.lr

```diff
@@ -0,0 +1,60 @@
+
+* We reworked the build system for Apple support packages to `separate the compilation of dependencies <https://github.com/beeware/cpython-apple-source-deps>`__ (like BZip2 and OpenSSL) from the compilation of Python itself. These libraries are updated infrequently, and on a completely different cadence to CPython itself, so treating them as re-usable resources rather than something that needs to be compiled at the same time as Python will significantly improve CPython build times.
+
+* We released `Briefcase 0.3.16 <https://pypi.org/project/briefcase/0.3.16/>`__, formalizing our support for Python 3.12, and adding the new support for dynamic libraries on iOS. This release also formally downgrades our support for AppImage, following extended problems in supporting AppImage as an output format.
+
+* We released `Rubicon ObjC 0.4.7 <https://pypi.org/project/rubicon-objc/0.4.7/>`__, formalising our support for Python 3.12.
+
+* We modified Briefcase to ensure that `macOS apps are always universal <https://github.com/beeware/briefcase/pull/1475>`__.
+
+* We added a `deep debug logging level <https://github.com/beeware/briefcase/pull/1470>`__ to Briefcase, and `simplified the output of many other commands to ensure that important warnings aren't lost <https://github.com/beeware/briefcase/pull/1503>`__.
+
+* We `added support for Undo and Redo operations on macOS text widgets <https://github.com/beeware/toga/pull/2151>`__.
+
+* We `added support for user-registered fonts on iOS and macOS <https://github.com/beeware/toga/pull/1399>`__.
+
+* We made some modifications to `improve Toga's memory management when creating and destroying <https://github.com/beeware/toga/pull/2066>`__.
+
+What's next?
+--------------
+
+Now that we've *finally* finished the widget audit, we can finally release Toga 0.4, and start using the foundation that we've spent the last 12 months building. 
+
+In November, we'll start looking at a technical limitation that the Toga audit has revealed on Android. A number of Toga's Android widgets aren't as fully featured as they could be, due to the fact that accessing the full API would require subclassing a Java base class. This is an area where Chaquopy has some limitations; so we're going to try and address these limitations.
+
+We'll also continue the work of upstreaming patches to CPython core. The PEP we've drafted is nearing a form where it can be submitted to the Steering Council for approval, but the drafting process has raised a couple of technical issues that need to be investigated. However, while we wait for formal acceptance of the PEP, we will continue working on building continuous integration for our iOS CPython builds, and begin preparing patches that can be merged into CPython.
```

Comment:
I'm not sure if the Java subclassing issue is the best thing to spend my time on now; let's discuss this tomorrow.

```suggestion
In November, we'll start the work of upstreaming iOS patches to CPython core. The PEP we've drafted is nearing a form where it can be submitted to the Steering Council for approval, but the drafting process has raised a couple of technical issues that need to be investigated. However, while we wait for formal acceptance of the PEP, we will continue working on building continuous integration for our iOS CPython builds, and begin preparing patches that can be merged into CPython.
```